crates/common/src/compile.rs

```diff
@@ -398,10 +398,16 @@ pub fn compile_target_with_filter(
     }
 }
 
+/// Id to map from the bytecode to the source file
+pub type FileId = u32;
+
+/// Map over artifcats contract sources name -> file_id -> (source, contract)
+pub type ContractSources = HashMap<String, HashMap<FileId, (String, ContractBytecodeSome)>>;
+
```

Comment:
please try to avoid these aliases, I really dislike them.

this should just be a new helper type.

Reply:
Can you please be extra explicit on what is a wrapper type ?
I found this alias convenient to have one place where it's defined and the reader can just goto definition and read the docs, instead of copy pasting them everywhere this is used and have this two-dimensional map.

Reply:
> Can you please be extra explicit on what is a new helper type ?

a new type that encapsulates this map.

this has a few advantages even if it requires more code,
1 type only
better debugging
custom helper functions

Reply:
I see, thanks